app/win/main_tl.js

```diff
@@ -1621,11 +1621,17 @@ var loadSavedWindowsFromData = (function(){
     ///     - bordered:<truthy> (default false) to mark windows with borders
     function loadSaveDataV1(data) {
         if(!data.tree) return false;
-        //log.info({'loadSaveDataV1':data});
+        const blankTabsRawUrl = ['about:blank', 'chrome://newtab'];
```

Comment:
I agree with the idea of a data structure listing the URLs.  Would you please use a hash instead of an array?  E.g., `{ 'about:blank': true ... }`.  For something this simple, I would like to avoid introducing a linear search.